src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/CreditorsAfterOneYearTransformerImpl.java

```diff
@@ -115,7 +110,7 @@ private void setCurrentPeriodOnApiModel(CreditorsAfterOneYear creditorsAfterOneY
         CurrentPeriod currentPeriod = new CurrentPeriod();
 
         if (creditorsAfterOneYear.getDetails() != null
-                && StringUtils.isBlank(creditorsAfterOneYear.getDetails())){
+                && StringUtils.isBlank(creditorsAfterOneYear.getDetails())) {
```

Comment:
This if / else block can be simplified to:
```
if (StringUtils.isBlank(creditorsAfterOneYear.getDetails())) {
       currentPeriod.setDetails(creditorsAfterOneYear.getDetails());
}
```
Removing the need for the else clause